brains/andalite.py

```diff
@@ -16,34 +17,34 @@ class Andalite(Sapiens):
     def __init__(self, game, args=None, server='ws://localhost:8765'):
         Sapiens.__init__(self, game, args)
         self.telepathy = Telepathy(server)
+        # get sync class for the currently loaded game.
+        game_sync_class = game_sync_classes.get_game_sync_class(game.emu.gameinfo['name'])
+        if game_sync_class != None:
```

Comment:
the python style is to say "if foo is not None" or "if foo is None" when comparing to None

Reply:
done